docs/index.md

```diff
@@ -1,37 +1,37 @@
-* [Bokeh app glider demo -- watch evolved gliders](https://mybinder.org/v2/gh/riveSunder/yuca/gecco_2022_pages?urlpath=/proxy/5006/gecco_bokeh_app) Note that this runs on [mybinder.org](https://mybinder.org), a great free cloud notebook service that can be a little slow. 
+* [Blog post](https://rivesunder.github.io/yuca_docs/evo_abio)
+* [Summary poster](https://raw.githubusercontent.com/riveSunder/yuca_docs/master/assets/halting_evo/pos237s1.pdf)
+* [Notebook](https://github.com/riveSunder/yuca_docs/blob/master/notebooks/halting_evo.ipynb) Try it on: [mybinder](https://mybinder.org/v2/gh/rivesunder/yuca_docs/master?labpath=notebooks%2Fhalting_evo.ipynb) -> [![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/rivesunder/yuca_docs/master?labpath=notebooks%2Fhalting_evo.ipynb), [colab](https://colab.research.google.com/github/rivesunder/yuca_docs/blob/master/notebooks/halting_evo.ipynb) -> [![Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/rivesunder/yuca_docs/blob/master/notebooks/halting_evo.ipynb) 
+* [Bokeh app glider demo -- watch evolved gliders](https://mybinder.org/v2/gh/riveSunder/yuca_docs/master?urlpath=/proxy/5006/gecco_bokeh_app) Note that this runs on [mybinder.org](https://mybinder.org), a great free cloud notebook service that can be a little slow. 
 
 <br><br>
 ####  Davis, Q, Tyrell and Bongard, Josh. Step Size is a Consequential Parameter in Continuous Cellular Automata. The 2022 Conference on Artificial Life. (2022). [https://doi.org/10.1162/isal_a_00526](https://doi.org/10.1162/isal_a_00526)
 
 * ArXiv -> [https://arxiv.org/abs/2205.12728](https://arxiv.org/abs/2205.12728)
 * [Repository](https://github.com/rivesunder/yuca) 
-* [Blog post](https://rivesunder.github.io/yuca/step_size) 
-* [Notebook](https://github.com/riveSunder/yuca/blob/master/notebooks/consequential_step_size.ipynb) Try it on: [mybinder](https://mybinder.org/v2/gh/rivesunder/yuca/master?labpath=notebooks%2Fconsequential_step_size.ipynb) -> [![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/rivesunder/yuca/master?labpath=notebooks%2Fconsequential_step_size.ipynb), [colab](https://colab.research.google.com/github/rivesunder/yuca/blob/master/notebooks/consequential_step_size.ipynb) -> [![Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/rivesunder/yuca/blob/master/notebooks/consequential_step_size.ipynb) 
-* [ALife 2022 slides](https://rivesunder.github.io/yuca/ss_slide_000)
+* [Blog post](https://rivesunder.github.io/yuca_docs/step_size) 
```

Comment:
Let's go ahead and revert the links to the gh-pages to point to `https://rivesunder.github.io/yuca` instead of yuca_docs. This should decrease navigation confusion while still offloading the larger assets storage to yuca_docs.